lib/improver/threshold.py

```diff
@@ -304,14 +309,12 @@ def process(self, input_cube):
             thresholded_cubes.append(cube)
 
         cube, = thresholded_cubes.concatenate()
-        # TODO: Correct when formal cf-standards exists
-        # Force the metadata to temporary conventions
+        # Use an spp__relative_to_threshold attribute, as an extension to the
+        # CF-conventions.
```

Comment:
I think this comment now belongs in the _add_threshold_coord function.

Reply:
I think I've corrected this now.